calc/src/mechanics/gen8.ts

```diff
@@ -396,17 +396,17 @@ function calculateModern(
   switch (move.name) {
     case 'Behemoth Bash':
     case 'Behemoth Blade':
-    case 'Dynamax Cannon',
-      basePower = move.bp * (['G-max', 'Dynamax'].indexOf(defender.name) !== -1) ? 2 : 1;
+    case 'Dynamax Cannon':
+      basePower = move.bp * (['Gmax', 'Dynamax'].indexOf(defender.name) !== -1 ? 2 : 1);
       description.moveBP = basePower;
       break;
     case 'Payback':
       basePower = turnOrder === 'LAST' ? 100 : 50;
       description.moveBP = basePower;
       break;
-    case 'Bolt Break':
+    case 'Bolt Beak':
     case 'Fishious Rend':
```

Comment:
also this is throwing an error which is contributing to the failing build

`src/mechanics/gen8.ts:408:5 - error TS7029: Fallthrough case in switch.`

not sure why

Reply:
im dumb